docs/gitbook/tutorials/apisix-progressive-delivery.md

```diff
@@ -51,7 +51,7 @@ kubectl create ns test
 Create a deployment and a horizontal pod autoscaler:
 
 ```bash
-kubectl apply -k https://github.com/fluxcd/flagger//kustomize/podinfo?ref=main
```

Comment:
The `//` is not a typo, this is how you tell kustomize to use that dir as root.

Reply:
Sorry about that then.
I tried it out with just one `/` and it worked just fine.
Furthermore I couldn't find anything in the docs regarding that: https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#kustomization-root

Would you mind posting a reference for me to have?

Reply:
See https://github.com/kubernetes-sigs/kustomize/blob/master/api/internal/git/repospec.go#L168-L170